includes/class-sensei-course.php

```diff
@@ -193,9 +201,108 @@ public function highlight_menu_item( $submenu_file ) {
 	}
 
 	/**
-	 * Add custom navigation to the admin pages.
+	 * Add showcase courses upsell page.
+	 *
+	 * @since $$next-version$$
+	 * @internal
+	 * @access private
+	 */
+	public function add_showcase_courses_upsell() {
+		add_submenu_page(
+			null,
+			__( 'Showcase Courses', 'sensei-lms' ),
+			__( 'Showcase Courses', 'sensei-lms' ),
+			'edit_courses',
+			self::SHOWCASE_COURSES_SLUG,
+			[ $this, 'showcase_courses_screen' ]
+		);
+	}
+
+	/**
+	 * Add showcase courses upsell screen.
+	 *
+	 * @since $$next-version$$
+	 * @internal
+	 * @access private
+	 */
+	public function showcase_courses_screen() {
+		// Enqueue styles.
+		Sensei()->assets->enqueue( 'sensei-showcase-upsell', 'css/showcase-upsell.css' );
+
+		$illustration = Sensei()->assets->get_image( 'showcase-courses-upsell-illustration.png' );
+		?>
+		<div class="wrap">
+			<?php
+				$screen = get_current_screen();
+				$tabs   = $this->get_course_custom_navigation_tabs();
+
+				$this->display_courses_navigation( $screen, $tabs );
+			?>
+			<div class="sensei-showcase-upsell">
+				<div class="sensei-showcase-upsell__content">
+					<h1 class="sensei-showcase-upsell__title"><?php esc_html_e( 'Showcase your course with Sensei Pro!', 'sensei-lms' ); ?></h1>
+					<p class="sensei-showcase-upsell__description"><?php esc_html_e( 'Gain visibility by promoting your post on Sensei Showcase Gallery page.', 'sensei-lms' ); ?></p>
+
+					<ul class="sensei-showcase-upsell__list">
+						<li class="sensei-showcase-upsell__list-item dashicons-before"><?php esc_html_e( 'Free traffic redirected to your site', 'sensei-lms' ); ?></li>
+						<li class="sensei-showcase-upsell__list-item dashicons-before"><?php esc_html_e( 'WooCommerce integration', 'sensei-lms' ); ?></li>
+						<li class="sensei-showcase-upsell__list-item dashicons-before"><?php esc_html_e( 'Schedule ‘drip’ content', 'sensei-lms' ); ?></li>
+						<li class="sensei-showcase-upsell__list-item dashicons-before"><?php esc_html_e( 'Set expiration date of courses', 'sensei-lms' ); ?></li>
+						<li class="sensei-showcase-upsell__list-item dashicons-before"><?php esc_html_e( 'Interactive Blocks', 'sensei-lms' ); ?></li>
+					</ul>
+
+					<div class="sensei-showcase-upsell__price-wrapper">
+						<span class="sensei-showcase-upsell__price"><?php esc_html_e( '$13 USD', 'sensei-lms' ); ?></span>
```

Comment:
I'm not sure! I checked the Groups upsell as reference and it's a hard-coded text.
But that's a good point. We could get it from the extensions endpoint, but it's only the annual price.

I added an issue for that in Sensei core, so we could improve it in the future. https://github.com/Automattic/sensei/issues/6658

Reply:
Interesting. I thought we always used dynamic pricing everywhere, especially because on frontend it is quite easy to do that using the extensions API as you mentioned. 

Let's leave it that way and maybe fix that in the issue, then. :)

Reply:
I'll add the issue to our backlog too. So we can fix it in this project if we have time. =)